### PR TITLE
wip : improve doc event hub

### PR DESCRIPTION
**Pull request by rombernier**

### CODE REVIEW

File: _shared_content/operations_center/integrations/event_hub.md [hunk 1]
```diff
@@ -16,60 +16,89 @@ From there, keep the relevant “Subscription ID” that will be used along the
 
 #### Step 2: Create Event Hub Namespace and associated Event Hub
```
[the remark this answers is not in the document]
Comment (rombernier) [reply _shared_content/operations_center/integrations/event_hub.md]: Done

File: _shared_content/operations_center/integrations/event_hub.md [hunk 2]
```diff
@@ -16,60 +16,89 @@ From there, keep the relevant “Subscription ID” that will be used along the
...
 
-Navigate to “Home”, “Event Hubs”. Create an Event Hub Namespace. 
-Select your Subscription and your Resource Group.
+Navigate to “Home”, “Event Hubs”. Create an Event Hub Namespace.   
+Select your Subscription and your Resource Group.  
+Click on create new if you want your event hub namespace in a new resource group.
```
[the remark this answers is not in the document]
Comment (rombernier) [reply _shared_content/operations_center/integrations/event_hub.md:21]: Done

File: _shared_content/operations_center/integrations/event_hub.md [hunk 3]
```diff
@@ -16,60 +16,89 @@ From there, keep the relevant “Subscription ID” that will be used along the
...
 
-Navigate to “Home”, “Event Hubs”. Create an Event Hub Namespace. 
-Select your Subscription and your Resource Group.
+Navigate to “Home”, “Event Hubs”. Create an Event Hub Namespace.   
+Select your Subscription and your Resource Group.  
+Click on create new if you want your event hub namespace in a new resource group.  
 
-When your Event Hub Namespace is created you can create an Event Hub inside 
-Navigate to “Home”, “Event Hubs”, “company-eventhubnamespace". Create an Event Hub. 
+Select the [pricing tier plan](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-quotas#basic-vs-standard-vs-premium-vs-dedicated-tiers) based on your utilisation: Standard or Premimum (Basic plan is to limited)
```
[the remark this answers is not in the document]
Comment (rombernier) [reply _shared_content/operations_center/integrations/event_hub.md]: Done

File: _shared_content/operations_center/integrations/event_hub.md
[the remark this answers is not in the document]
Comment (rombernier) [reply _shared_content/operations_center/integrations/event_hub.md:1]: Done

**Review by CharlesLR-sekoia — APPROVED** [review]

**Pull request merged**